app/assets/stylesheets/application.scss

```diff
@@ -47,10 +47,42 @@
 }
 
 .navbar-skinny {
-  min-height: 1.5em;
+  min-height: 2.5em;
 }
 
 .add-doc {
-  width: 3em;
+  width: 1.5em;
+  height: 1.5em;
+  padding-right: .5rem;
   display: inline-block;
 }
+
+.btn-secondary {
+  color: $white;
+}
+
+.view-btn {
+  margin-left: .75rem;
```

Comment:
what's the purpose of these?

Reply:
Cos Ross wants to shift it inline with the text. Open to sugestions :smile:

Reply:
if these classes are all doing the same thing, you could just have one class and give it to all of them - like `shifted-btn` or something

Reply:
But the other button is the opposite & needs a margin-right instead :frowning_face:

Reply:
`shifted-left-btn`, `shifted-right-btn`